src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.ts

```diff
@@ -117,3 +121,27 @@ export class RollCallComponent implements OnInit {
     return this.currentIndex >= this.entries?.length;
   }
 }
+
+@DatabaseEntity("RollCallConfig")
```

Comment:
This decorator should only be used when a class has to be saved to the database which I think is not necessary in this case.

Reply:
Yes, removed.